templates/CRM/common/importProgress.tpl

```diff
@@ -0,0 +1,77 @@
+{*
+ +--------------------------------------------------------------------+
+ | CiviCRM version 4.7                                                |
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC (c) 2004-2017                                |
+ +--------------------------------------------------------------------+
+ | This file is a part of CiviCRM.                                    |
+ |                                                                    |
+ | CiviCRM is free software; you can copy, modify, and distribute it  |
+ | under the terms of the GNU Affero General Public License           |
+ | Version 3, 19 November 2007 and the CiviCRM Licensing Exception.   |
+ |                                                                    |
+ | CiviCRM is distributed in the hope that it will be useful, but     |
+ | WITHOUT ANY WARRANTY; without even the implied warranty of         |
+ | MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.               |
+ | See the GNU Affero General Public License for more details.        |
+ |                                                                    |
+ | You should have received a copy of the GNU Affero General Public   |
+ | License and the CiviCRM Licensing Exception along                  |
+ | with this program; if not, contact CiviCRM LLC                     |
+ | at info[AT]civicrm[DOT]org. If you have questions about the        |
+ | GNU Affero General Public License or the licensing of CiviCRM,     |
+ | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
+ +--------------------------------------------------------------------+
+*}
+{literal}
+<script type="text/javascript">
+CRM.$(function($) {
+  var loop;
+  $("form#Preview").on('submit', function (e) {
+    verify();
+  });
+  function setIntermediate() {
+    var dataUrl = "{/literal}{$statusUrl}{literal}";
+    $.getJSON(dataUrl, function(response) {
+      var dataStr = response.toString();
+      var result  = dataStr.split(",");
+      $("#intermediate").html(result[1]);
+      $("#importProgressBar .ui-progressbar-value").show();
+      if (result[0] < 100) {
+        $("#importProgressBar .ui-progressbar-value").animate({width: result[0] + "%"}, 500);
+        $("#status").text(result[0] + "% Completed");
+      }
+    });
+  }
+
+  function verify() {
+    if (!confirm('Backing up your database before importing is recommended, as there is no Undo for this. {/literal}{ts escape='js'}Are you sure you want to Import now{/ts}{literal}?')) {
```

Comment:
Please fix the `{ts}` usage in this string:  the part "Backing up database before importing ..." should be in a `ts`.

Ex: `if (!confirm('{/literal}{ts escape='js'}Backuping up ....{/ts} {ts escape='js'}Are you sure ...?{/ts}{literal}'))`

Also: `{ts escape='js'}Are you sure you want to Import now{/ts}{literal}?'`  -> the interrogation should be inside the `{ts}`.